internal/commands/buildpack_package.go

```diff
@@ -76,6 +76,15 @@ func BuildpackPackage(logger logging.Logger, cfg config.Config, client Buildpack
 			}
 
 			name := args[0]
+			if flags.Format == pack.FormatFile {
+				switch ext := filepath.Ext(name); ext {
+				case pack.CNBExtension:
+				case "":
+					name += pack.CNBExtension
+				default:
+					logger.Warnf("%s is not a valid extension for a Buildpackage. Buildpackages must have a %s extension", style.Symbol(ext), style.Symbol(pack.CNBExtension))
```

Comment:
We don't use the term `buildpackages` in the Long description, so we probably should standardize what we are calling it – either a `packaged buildpack`, or a `buildpackage`.

Reply:
Renamed to packaged buildpack.